test/fail_compilation/fail6795.d

```diff
@@ -1,24 +1,17 @@
 // https://issues.dlang.org/show_bug.cgi?id=6795
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail6795.d(12): Error: cannot modify constant `0`
-fail_compilation/fail6795.d(13): Error: `[0][0]` is not an lvalue and cannot be modified
----
-*/
 
 void main() {
     enum int[] array = [0];
     array[0]++;
+    --array[0];
     array[0] += 3;
```

Comment:
This would be 'valid' code now, each line mutating its own little GC array. @Geod24

Reply:
It doesn't really shock me. I always explain `enum` as "better `#defines`". A linter could catch this.